transform/binary-plugin/binary_plugin.go

```diff
@@ -68,17 +68,17 @@ func (b *BinaryPlugin) Run(request transform.PluginRequest) (transform.PluginRes
 	p := transform.PluginResponse{}
 
 	out, errBytes, err := b.commandRunner.Run(request, b.log)
-	if err != nil {
-		b.log.Errorf("error running the plugin command")
-		return p, fmt.Errorf("error running the plugin command: %v", err)
-	}
-
 	if len(errBytes) != 0 {
 		logs := strings.Split(string(errBytes), "\n")
 		for _, line := range logs {
 			b.log.Debug("Plugin Log line: ", line)
 		}
 	}
+	if err != nil {
```

Comment:
This is to make sure that we get stderr from the plugin run logged even when the plugin fails. I was debugging a failed plugin run but stderr was being swallowed because we only added the "Plugin log line" debug statements if the plugin run *didn't* error out. I just moved the debug logging block up to happen before the error return.